### PR TITLE
elpa: add 2018.05.001.rc1 and 2017.11.001

### PR DESCRIPTION
**Pull request by davydden**
absolutely untested as it simply does not configure / build on macOS 😞

### CODE REVIEW

File: var/spack/repos/builtin/packages/elpa/package.py [hunk 1]
```diff
@@ -31,6 +31,8 @@ class Elpa(AutotoolsPackage):
     homepage = 'http://elpa.mpcdf.mpg.de/'
     url = 'http://elpa.mpcdf.mpg.de/elpa-2015.11.001.tar.gz'
 
+    version('2018.05.001.rc1', 'ccd77bd8036988ee624f43c04992bcdd')
+    version('2017.11.001', '4a437be40cc966efb07aaab84c20cd6e')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/elpa/package.py]: Can you make this version preferred?

**Comment by davydden** [comment]
@adamjstewart ping.

**Comment by adamjstewart** [comment]
Did previous versions build on macOS? I feel uncomfortable about adding new versions if they don't even build.

**Comment by davydden** [comment]
> Did previous versions build on macOS?

No

**Pull request merged**